components/more_direct_channels/list/list.tsx

```diff
@@ -4,12 +4,11 @@
 import React, {useCallback, useMemo} from 'react';
 import {FormattedMessage, useIntl} from 'react-intl';
 
-import {UserProfile} from '@mattermost/types/users';
-
 import MultiSelect from 'components/multiselect/multiselect';
-
 import Constants from 'utils/constants';
 
+import {UserProfile} from '@mattermost/types/users';
```

Comment:
@hmhealey i dont know what changed but i am getting is eslint errors lately since yesterday to place `@mattermost/types/users` below 🤔

Reply:
@M-ZubairAhmed Have you run a clean and reinstalled node_modules? I got an error like that before when it couldn't find the imports since it didn't realize they were local

Reply:
This is actually the opposite of what I'd expect since I would've thought ESLint would treat these as external for sorting. Perhaps I'm wrong though?